### ISSUE TITLE
Can't connect to rabbit mq server

### ISSUE THREAD

**Issue by shreyas-algo** [kegaretail/react-native-rabbitmq#3]
First of all I think `const` is misspelled as `cont` in the README and `system_exchange` should be `exchange`

Having changed these, when I try to connect to rabbit mq server using the config properties, I get :
`Connection {rabbitmqconnection: Object, callbacks: Object, connected: false}` as the connection object where connection = : let connection = new Connection(config); 

I have referred https://www.rabbitmq.com/tutorials/tutorial-one-javascript.html to create rabbit mq server on node side

I am sure of the config properties because using same I can connect to rabbit mq from a separate node server. Tried to add loggers in both `connection.on('error')` & in `connection.on('connected')` but did not get any log

Unable to understand where the issue is.

Please help

**Comment by timhonders** [comment]
Hi,

Tnx, for the typo's,

Witch OS (IOS/Android) are u trying

**Comment by shreyas-algo** [comment]
Hi

Trying with Android

**Comment by timhonders** [comment]
Can u make an adb log?

**Comment by SupaMario123** [comment]
I have the same problem. 
Get the following error:

` description: 'failed to connect to /127.0.0.1 (port 5672) from /127.0.0.1 (port 55327) after 60000ms: isConnected failed: ECONNREFUSED (Connection refused)',

Device is an Android. ` 

Turned out that the following Exception is thrown:

	2019-05-05 15:02:50.491 14594-15039/? E/RabbitMqConnection: com.facebook.react.bridge.NoSuchKeyException: ssl

When i comment out the following code it doesnt work at all. 

```
        try {
            if (this.config.getBoolean("ssl")) {
                this.factory.useSslProtocol();
            }
        } catch(Exception e) {
            Log.e("RabbitMqConnection", e.toString());
        }

```


Any ideas? @timhonders would be very helpful :)

**Comment by timhonders** [comment]
if u try

const config = {
	host:'',
	port:5672,
	username:'user',
	password:'password',
	virtualhost:'vhost',
	ttl: 10000 // Message time to live,
	ssl: false
}

**Comment by timhonders** [comment]
It should be fixed in 0.6.1
